src/components/navbar/Navbar.tsx

```diff
@@ -22,28 +22,24 @@ export default function navBar() {
       }
     }
   `);
+
   const { username } = useContext(UserContext);
 
-  return !username ? (
+  return (
     <Navbar variant="dark" className="shadow">
-      <Navbar.Brand className="pl-md-5 mt-2" style={{alignSelf: 'center'}}>
+      <Brand className="pl-md-5 mt-2">
         <Link to="/">
-          <Img fixed={data.file.childImageSharp.fixed} alt="Callibrity Logo" />
+          <Img fixed={callibrityLogo} alt="Callibrity Logo" />
         </Link>
-      </Navbar.Brand>
-    </Navbar>
-  ) : (
-    <Navbar variant="dark" className="shadow">
-      <Navbar.Brand className="pl-md-5 mt-2">
-        <Link to="/">
-          <Img fixed={data.file.childImageSharp.fixed} alt="Callibrity Logo" />
-        </Link>
-      </Navbar.Brand>
-      <Nav className="ml-auto pr-md-5">
-        <SearchBar style={{ marginRight: '10px' }} />
+      </Brand>
+      {
+       username && 
```

Comment:
is this username && syntax keeping the links and such from showing up before logging in since you got rid of the !username conditional above?

Reply:
Yes,  "boolean && Component" will render the component if the boolean is true, or render null if the boolean is false